### ISSUE TITLE
Error: Cannot find module 'any-promise'

### ISSUE THREAD

**Issue by NekR** [mozilla/sign-addon#111]
```sh
NekR@MacBook:~/...$ gulp ...
Error: Cannot find module 'any-promise'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/.../node_modules/sign-addon/dist/sign-addon.js:227:19)
    at __webpack_require__ (/.../node_modules/sign-addon/dist/sign-addon.js:22:30)
    at Object.<anonymous> (/.../node_modules/sign-addon/dist/sign-addon.js:163:16)
    at __webpack_require__ (/.../node_modules/sign-addon/dist/sign-addon.js:22:30)
    at Object.<anonymous> (/.../node_modules/sign-addon/dist/sign-addon.js:57:12)
    at __webpack_require__ (/.../node_modules/sign-addon/dist/sign-addon.js:22:30)
```

**Comment by NekR** [comment]
Also I cannot build this locally, zillion of errors like missing files in different modules. Here is the gist with logs, if it helps: https://gist.github.com/NekR/ba9e38ecd41eef9009aa628684a48dcd

**Comment by NekR** [comment]
```
NekR@MacBook:~/Projects/sign-addon$ node -v
v4.2.2
```

**Comment by NekR** [comment]
Okay, fixed npm version by installing `any-promise` and `thenify-all` in my project. Seems like you hate to add them to your `dependencies`.

**Comment by kumar303** [comment]
> Seems like you hate to add them to your dependencies.

I would hate to do that, yes, because they aren't direct dependencies. They are sub-dependencies, a relationship handled by npm during the install process. Thus, what you are experiencing is an npm bug. I can reproduce it with npm 2.14.7 but it's fixed in 3.10.2.

Upgrade and rebuild like this:
````
cd /path/to/sign-addon
npm install -g npm
rm -fr node_modules
npm install
````

Thanks for the bug report, I updated the docs to warn about npm bugs: https://github.com/mozilla/sign-addon/commit/25bfe132cb75129cb44832bb5aa3d9d3d52b2ee6
